gtfs/spec/en/reference.md

```diff
@@ -43,8 +44,8 @@ This specification defines the following files along with their associated conte
 |  [routes.txt](#routestxt) | **Required** | Transit routes. A route is a group of trips that are displayed to riders as a single service. |
 |  [trips.txt](#tripstxt)  | **Required** | Trips for each route. A trip is a sequence of two or more stops that occurs at specific time. |
 |  [stop_times.txt](#stop_timestxt)  | **Required** | Times that a vehicle arrives at and departs from individual stops for each trip. |
-|  [calendar.txt](#calendartxt)  | **Required** | Dates for service IDs using a weekly schedule. Specify when service starts and ends, as well as days of the week where service is available. |
-|  [calendar_dates.txt](#calendar_datestxt)  | Optional | Exceptions for the service IDs defined in the [calendar.txt](#calendartxt) file. If [calendar.txt](#calendartxt) includes ALL dates of service, this file may be specified instead of [calendar.txt](#calendartxt). |
+|  [calendar.txt](#calendartxt)  | **Conditionally required** | Dates for service IDs using a weekly schedule. Specify when service starts and ends, as well as days of the week where service is available. |
+|  [calendar_dates.txt](#calendar_datestxt)  | **Conditionally required** | Exceptions for the service IDs defined in the [calendar.txt](#calendartxt) file. If [calendar.txt](#calendartxt) includes ALL dates of service, this file may be specified instead of [calendar.txt](#calendartxt). |
```

Comment:
Yes for the typo in the spec.

But No for the second proposition. We already have a sentence for that IMHO. Do you want to replace the sentence because you find is badly written (If [calendar_dates.txt](#calendar_datestxt) includes ALL dates of service, this file may be specified instead of [calendar.txt](#calendartxt).), or do you want to add also your sentence after it

Reply:
Reading it again I suppose we could use `If [calendar.txt](#calendartxt) is omitted then this file is required and must contain all dates of service.` in place of the previous sentence (vs. in addition to).  If the field is labeled `Conditionally required` IMHO we should clearly state when the field is required (using that language) in the Details section which I why I proposed this language.